sigs.yaml

```
@@ -2717,7 +2717,15 @@ workinggroups:
     - github: zacharysarah
       name: Zach Corleissen
       company: Linux Foundation
-  meetings: []
+  meetings:
+  - description: Regular WG Meeting
+    day: Monday
+    time: "10:30"
+    tz: PT (Pacific Time)
+    frequency: monthly - second Monday of month
+    url: https://zoom.us/j/91522666403?pwd=R0FpT3pCZm84c2ZYT2xRTjI5emc1UT09
```

```suggestion
    url:  https://zoom.us/j/91522666403?pwd=WnRSNlNhNXhDWkR2ZU9ydGpsNWxtZz09
```

Just updated it for the new passcode

---

Since things involve generation here, I'm amending the commit rather than using the GH UI to accept this.  Fixed commit coming in just a moment..